### PR TITLE
tokenin*AnyKV filters should be able to check same keys

### PR DESCRIPTION
**Pull request by szuecs**
add failing tests that should work after fixing https://github.com/zalando/skipper/issues/1044

- [x] add failing tests
- [x] add implementation
- [x] fix documentation

Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: docs/reference/filters.md [hunk 1]
```diff
@@ -471,6 +471,7 @@ Examples:
 
 ```
 oauthTokeninfoAnyKV("k1", "v1", "k2", "v2")
```
Comment (tkrop) [inline docs/reference/filters.md]: The difference is very difficult to see when the two lines look that similar. Let's use on example line with both or change the example values to make this more visible.

**Comment by tkrop** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**